.pre-commit-config.yaml

```diff
@@ -33,20 +33,27 @@ repos:
         files: backend/
 
   - repo: https://github.com/pycqa/flake8
-    rev: 4.0.1
+    rev: 5.0.4
     hooks:
       - id: flake8
         name: flake8-backend
         files: backend/
-        args: ['--config', 'backend/setup.cfg']
+        args: [ '--config', 'backend/setup.cfg' ]
```

Comment:
can we have [consistency](https://github.com/HungryBroke/HungryBroke/pull/12/files#diff-c0d9756ec1e666a5c4b1ab6fa27d5662156838384a67802e705e4de44d6ab119L9) in the spacing?